sql/core/src/main/scala/org/apache/spark/sql/internal/SessionState.scala

```diff
@@ -159,6 +161,13 @@ class SessionResourceLoader(session: SparkSession) extends FunctionResourceLoade
     }
   }
 
+  protected def resolveJars(path: URI): Seq[String] = {
```

Comment:
Although this class is `SessionResourceLoader `, `resolveJars` looks like more a utility function. Do we need to add here as `protected def`?

Reply:
> Although this class is `SessionResourceLoader `, `resolveJars` looks like more a utility function. Do we need to add here as `protected def`?

remove `protect` since user or other feature can use this method